### PR TITLE
Clean up delete job a bit

### PR DESCRIPTION
**Pull request by rkondratowicz**
- fix formatting
- fix PMD issue
- make logging more concise

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java [hunk 1]
```diff
@@ -88,65 +88,41 @@ private void processCompleteFiles(CloudBlobContainer container) {
...
-    @SuppressWarnings("PMD.DataflowAnomalyAnalysis")
     private boolean tryProcessCompleteEnvelope(CloudBlobContainer container, Envelope envelope) {
-        boolean deleted = false;
+        boolean deleted;
+
+        String loggingContext = "File name: " + envelope.getZipFileName() + ", Container: " + container.getName();
 
         try {
-            CloudBlockBlob cloudBlockBlob = container.getBlockBlobReference(envelope.getZipFileName());
+            log.info("Deleting file. " + loggingContext);
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java:97]: can be log specific parameterised :wink:

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java [hunk 2]
```diff
@@ -88,65 +88,41 @@ private void processCompleteFiles(CloudBlobContainer container) {
         );
     }
 
-    @SuppressWarnings("PMD.DataflowAnomalyAnalysis")
     private boolean tryProcessCompleteEnvelope(CloudBlobContainer container, Envelope envelope) {
-        boolean deleted = false;
+        boolean deleted;
+
+        String loggingContext = "File name: " + envelope.getZipFileName() + ", Container: " + container.getName();
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java:94]: `String.format` or like this ^? I don't mind really

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java (hunk 1 above)
Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java]: fixed

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java (hunk 2 above)
Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java:94]: won't fit into one line ;)

**Pull request merged**